Tella/Supporting Files/TellaConfig.xcconfig

```diff
@@ -0,0 +1,12 @@
+//
+//  TellaConfig.xcconfig
```

Comment:
I think it was a mistake to add the `TellaConfig.xcconfig` file before merging the PR that includes the `.gitignore` file.

Reply:
Could you delete this file and regenerate the keys